src/Routes/Devices/DeviceTable.js

```diff
@@ -138,7 +141,10 @@ const createRows = (
         {
           title: hasLinks
             ? createLink({
-                pathname: `${deviceLinkBase}/${DeviceUUID}`,
+                pathname:
+                  currentInventoryPath === '/edge'
+                    ? `${DeviceUUID}`
+                    : `${deviceLinkBase}/${DeviceUUID}`,
```

Comment:
maybe the problem is that we are not passing "deviceLinkBase" to createRows function ?